### PR TITLE
Add SelectivityVector::toString() method

### PR DESCRIPTION
**Pull request by mbasmanova**
The toString() method is helpful when developing or debugging. It allows to print a summary of all the selected rows as well as row numbers of a few first selected rows.

### CODE REVIEW

File: velox/vector/SelectivityVector.cpp [hunk 1]
```diff
@@ -27,4 +27,29 @@ SelectivityVector SelectivityVector::empty(vector_size_t size) {
...
+
+  VELOX_CHECK_GE(maxSelectedRowsToPrint, 0);
+
+  std::stringstream out;
+  out << selectedCnt << " out of " << size() << " rows selected between "
+      << begin() << " and " << end();
+
+  if (selectedCnt > 0 && maxSelectedRowsToPrint > 0) {
+    out << ": ";
+    int cnt = 0;
```
Comment (majetideepak) [inline velox/vector/SelectivityVector.cpp:42]: We should probably use something like this `bool firstValidEncountered = true;` and set `firstValidEncountered = false;` after the first value to make the intent explicit.

**Review by majetideepak — COMMENTED** [review]
Changes look good.

File: velox/vector/SelectivityVector.cpp (hunk 1 above)
Comment (majetideepak) [reply velox/vector/SelectivityVector.cpp:42]: Never mind. I missed that `cnt` is also used to limit the output. We don't need another variable for the delimiter.

Comment (mbasmanova) [reply velox/vector/SelectivityVector.cpp:42]: @majetideepak Thank you for reviewing, Deepak.

**Pull request closed without merging**